src/Core/AndroidNative/maui/src/main/java/com/microsoft/maui/glide/font/FontModel.java

```diff
@@ -62,4 +73,16 @@ public ObjectKey getCacheKey()
                 + String.valueOf(textSize)
                 + typeface.toString());
     }
+
+    private String getGlyphHex() {
+        if (glyph == null) {
+            return null;
+        }
+
+        byte[] ba = glyph.getBytes(StandardCharsets.UTF_16);
+        StringBuilder str = new StringBuilder();
+        for(int i = 2 /* Skip (BOM) */; i < ba.length; i++)
+            str.append(String.format("%02x", ba[i]));
+        return str.toString();
+    }
```

Comment:
It's hard to know if this code is correct... Should we make it `public` and `static` and at least write 1 or 2 C# device tests that call it?

Reply:
<img width="410" alt="image" src="https://github.com/user-attachments/assets/31e45cc5-a287-4ea7-99a9-b0fd0428789c">